index.html

```diff
@@ -0,0 +1,139 @@
+          <li><a href="#"><img class="sm" src="./images/twitter.png" alt="Twitter Link"></a></li>
+        </ul>
+        <a class="language" href="https://summit.creativecommons.org/"><p class="lang-text">English </p></a>
+      </div>
+
+      <nav class="main-nav">
+        <div class="hamburger"><img class="hamburger-icon" src="./images/bars-solid.svg" alt=""></div>
+        <div class="school-logo" ><a href="index.html"><img class="school-logo-img" src="./images/School-logo.jpeg" alt="Royal School Logo"></a></div>
+        <div class="nav-menu">
+          <div class="close-icon"><img src="images/close.png" alt="close icon"></div>
+            <ul class="nav-links">
+              <li><a href="about.html">About</a></li>
+              <li><a href="#">Program</a></li>
+              <li><a href="#">Application Forms</a></li>
+              <li><a href="#">Partners</a></li>
+              <li><a href="#">School News</a></li>
+              <li class="presentation"><a href="#">Presentation</a></li>
+            </ul>
+        </div>
+      </nav>
+    </header> 
+
+    <section id="main">
+
+      <section id="title">
+        <div class="intro-text">
+          <img class="chess-deco" src="./images/chessboard.jpg" alt="">
+          <h5 class="slogan">"Empowering young royals to rule their future!"</h5>
+          <h2 class="rss" >ROYAL SECONDARY SCHOOL</h2>
+        </div>
+        <div class="intro-desc">
+          <p class="mission">Empowering Excellence: At Royal School, we are dedicated to providing top-quality 
+            education and training through modern, rigorous methods in a safe and supportive 
+            environment that delivers exceptional value.
+          </p>          
+        </div>
+        <img class="circle" src="./images/BG-design-color.png" alt="">
+        <div class="school-dets">
+          <p class="timetable">MON - FRI: 07:30 - 15:30</p>
+          <p class="location">@ Buteko Avenue, Town Center, Ndola, Zambia</p>
+        
+        </div>
+      </section>
+    </section>
+
+    <section id="features">
+      <div class="school-values-title">
+        <p>Main School Values</p>
+      </div>
+      <div class="title-line"></div>
+      <div class="school-features">
+
+        <div class="school-feature">
+          <div class="feature-icon"><img src="./images/fairness.png" alt=""></div>
+          <div class="feature-item"><p>Fairness</p></div>
+          <div class="feature-desc"><p>We treat our pupils, employees and all stakeholders in a fair and respectful manner.</p></div>
+        </div>
+        
+        <div class="school-feature">
+          <div class="feature-icon"><img src="./images/progress.png" alt=""></div>
+          <div class="feature-item"><p>Progress</p></div>
+          <div class="feature-desc"><p>We strive for continuous improvement, advancing our capacity and facilities to serve future generations.</p></div>
+        </div>
+
```

Comment:
- [ ] `Required`
- [ ] Amazing work so far 👏 
- [ ] However there is a few thing 🤏 you to adjust.
- [ ] Kindly use the background used the original design.
- [ ] please see screenshots below 🥂 

`your design`
![image](https://user-images.githubusercontent.com/66230306/219821513-1a1b6eab-6c79-4823-a6d9-68512f37533c.png)

`origin design`
![image](https://user-images.githubusercontent.com/66230306/219821620-4e905e38-ee85-4163-8f6d-0fa42f7b0ec7.png)